bc_obps/common/fixtures/dashboard/bciers/external.json

```diff
@@ -16,42 +16,88 @@
               {
                 "title": "Select an Operator",
                 "href": "/administration/select-operator",
-                "condition": {
-                  "api": "registration/v2/user-operators/current/operator",
-                  "field": "error",
-                  "operator": "exists",
-                  "value": true
-                }
+                "conditions": [
+                  {
+                    "api": "registration/v2/user-operators/current/operator",
```

Comment:
Nothing to suggest here, just sharing my thoughts based on our previous conversation. We are calling some of these APIs multiple times to check the same condition when the user loads the dashboard page.
![Screenshot 2024-10-17 at 2 46 13 PM](https://github.com/user-attachments/assets/c8bd1472-efb5-4879-848c-f3034c1ede82)

At some point, we may need to implement caching to reduce the number of database calls when a user opens the dashboard page (or similar pages).

An example of caching in Django-Ninja 👇 
https://django-ninja.dev/whatsnew_v1/?h=cache#decorators:~:text=%40decorate_view(cache_page(5))

I was curious and tested that, and here are the results: 🤩 
![Screenshot 2024-10-17 at 4 02 16 PM](https://github.com/user-attachments/assets/476477b0-835c-4acf-9b84-91c5fb4b4e64)